pyproject.toml

```diff
@@ -236,7 +236,7 @@ legacy_tox_ini = """
 # Remember to start celery workers to run celery tests, e.g.
 # celery --app=superset.tasks.celery_app:app worker -Ofair -c 2
 [testenv]
-basepython = python3.10
+basepython = python3.11
```

Comment:
Just to double check: if this version change were to affect any other tests or logic, it should fail the GH Actions for this PR as well, right?

Reply:
Ah, I need to revert this, it was for testing locally. Sorry.

Reply:
Hey @betodealmeida I believe this should be good to be approved as soon as you revert this change 🙌